auto_round/export/export_to_autoround/export.py

```diff
@@ -166,6 +166,7 @@ def save_quantized_as_autoround(output_dir, inplace=True, backend="auto_round:ex
 
     model = kwargs["model"]
     multimodal = kwargs["multimodal"]
+    safe_serialization = True if 'safe_serialization' not in kwargs.keys() else  kwargs["safe_serialization"]
```

Comment:
why need this

Reply:
if safe serialization  has no effect for other models, we could use it as the default in the future